content/docs/user-guide/components/reference/atom/stars.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Stars Component
+linktitle: Stars
+description: 'Open 3D Engine (O3DE) Stars component reference.'
+toc: true
+---
+
+The **Stars** component provides physically-based animated resolution-independant distant stars.
+
+Stars are rendered as billboards containing a white center that fades out to the star color on the edges.  The billboards are projected onto the far plane, and the brightness of each changes over time so they "twinkle". The orientation of the entire star field is controlled by the **Transform** component of the entity the Stars Component is on, so you can easily rotate the star field to simulate the rotation of the earth.
+
+![A](/images/user-guide/components/reference/atom/stars/stars.png)
+
+## Provider ##
+
+[Stars Gem](/docs/user-guide/gems/reference/rendering/stars/)
+
+## Properties
+
+![stars-component-base-properties](/images/user-guide/components/reference/atom/stars/stars-base-properties-ui.png)
+
+| Property | Description | Values | Default |
+|-|-|-|-|
+| **Exposure** | Specifies the exposure to use when rendering the stars in the scene so you can control how bright the stars are. | `0.0` to `32.0` | `1.0` |
+| **Radius Factor** | Specifies the factor to multiply every star width and height by so you can set the size of stars. | `0.0` to `64.0`  | `7.0`  |
```

Comment:
```suggestion
| **Radius Factor** | Specifies the factor to multiply every star width and height by so you can set the size of stars. | `0.0` to `64.0`  | `7.0`  |
| **Stars Asset** | The Stars binary data file to use for every star position, color and brightness. | `.stars` Asset | `default.stars` |
```
Move property to match the order of the component's UI.  Add source asset extension as expected value.